packages/elements/src/react/common/link.tsx

```diff
@@ -0,0 +1,42 @@
+import { useClerk } from '@clerk/shared/react';
+import { useClerkRouter } from '@clerk/shared/router';
+import type { ClerkOptions } from '@clerk/types';
+import React from 'react';
+
+type Destination = 'sign-in' | 'sign-up';
+export interface LinkProps extends Omit<React.HTMLAttributes<HTMLAnchorElement>, 'children'> {
+  navigate: Destination;
+  children: React.ReactNode | ((url: string) => React.ReactNode);
```

Comment:
Can we do { url } in case we need to add more ?

Reply:
I am fine with that. Do we keep the prop name as `url`?

Reply:
updated ✅